unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/SkyboxController.cs

```diff
@@ -44,6 +44,9 @@ public class SkyboxController : IPlugin
         private int syncCounter = 0;
         private int syncAfterCount = 10;
 
+        // Report to kernel
+        private TimeReporter timeReporter = new TimeReporter();
```

Comment:
I would extract an interface for `TimeReporter` so we can mock it.

Reply:
done